### DISCUSSION TITLE
Integrazione Robot aspirapolvere in HA attraverso Broadlink

### DISCUSSION THREAD

**Discussion by jumpig** [indomus/forum#1313] [HUB: Home Assistant]
Ciao a tutti,
        è il mio primo post qui e spero di non fare errori.
Ho provato a seguire il [tutorial specifico](https://indomus.it/progetti/domotizzare-un-robot-aspirapolvere-non-domotico-con-broadlink-e-home-assistant/) ma al momento non riesco a far funzionare il telecomando, forse potete aiutarmi a scoprire dove ho sbagliato.
Al momento l'unica evidenza che ho è un messaggio su NODE-RED che riporta un "No connection" nel momento in cui premo il pulsante di azione su HA
![image](https://user-images.githubusercontent.com/1162028/149315717-dc032117-aee5-4428-9d7e-fdcfd7010476.png)

Lato NODE-RED non vedo altri LOG, lato HA ho solo questa entry nel log che **potrebbe** essere riferita a questo setup:
![image](https://user-images.githubusercontent.com/1162028/149316124-7c5c12e7-5ac9-4276-a11d-727386df1c12.png)

Avete qualche idea per eseguire un debug e capire dove andare a sbattere la testa?
Fatemi sapere se può esservi utile qualche pezzo dei codici di configurazione.
Grazie!

**Comment by indomus** [comment]
> Ciao a tutti, è il mio primo post qui e spero di non fare errori. Ho provato a seguire il [tutorial specifico](https://indomus.it/progetti/domotizzare-un-robot-aspirapolvere-non-domotico-con-broadlink-e-home-assistant/) ma al momento non riesco a far funzionare il telecomando, forse potete aiutarmi a scoprire dove ho sbagliato. Al momento l'unica evidenza che ho è un messaggio su NODE-RED che riporta un "No connection" nel momento in cui premo il pulsante di azione su HA ![image](https://user-images.githubusercontent.com/1162028/149315717-dc032117-aee5-4428-9d7e-fdcfd7010476.png)
> 
> Lato NODE-RED non vedo altri LOG, lato HA ho solo questa entry nel log che **potrebbe** essere riferita a questo setup: ![image](https://user-images.githubusercontent.com/1162028/149316124-7c5c12e7-5ac9-4276-a11d-727386df1c12.png)
> 
> Avete qualche idea per eseguire un debug e capire dove andare a sbattere la testa? Fatemi sapere se può esservi utile qualche pezzo dei codici di configurazione. Grazie!

C'erano svariati casini nella guida, ti chiediamo scusa. Ora è aggiornata: devi rifare un passo (creazione token lunga vita) e configurare i nodi che chiamano i servizi Home Assistant in modo differente. Assicurati di avere l'ultima versione di Node-RED e di `node-red-contrib-home-assistant-websocket`.
